docker/osd-dev/README.md

```diff
@@ -43,25 +59,48 @@ the versions of OpenSearch which will be included into a Wazuh
 version.
 
 `OpenSearch` supported versions:
+
 - 1.2.4
+- 2.1.0
+- 2.2.1
 - 2.3.0
 - 2.4.0
 - 2.4.1
+- 2.6.0
+- 2.8.0
+- 2.9.0
+- 2.10.0
+- 2.11.0
+- 2.11.1
+- 2.12.0
+- 2.13.0
 
 `OpenSearch Dashboards` supported versions:
+
 - 1.2.0
+- 2.1.0
+- 2.2.1
 - 2.3.0
 - 2.4.0
 - 2.4.1
+- 2.6.0
+- 2.8.0
+- 2.9.0
+- 2.10.0
+- 2.11.0
+- 2.11.1
+- 2.12.0
+- 2.13.0
```

Comment:
This may not be necessary, as there is not a version of wazuh with every version of opensearch, so in theory we do not support all versions I believe.